about/principles.qmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Values and principles"
+---
+
+Any human creation is influenced by our values and perspectives of the
+world around us. We can let these values guide us in our work on
+Seedcase Project unconsciously and implicitly, without examining,
+critiquing, and evaluating them. Or we can explicitly and consciously
+select, describe, and explain the values we want to be guided by.
+
+We want to be explicit and conscious about what, how, and importantly,
+why we are doing what we decide to do. Being explicit will also help
+focus our work and (hopefully) make us more efficient and effective.
+
+The values and principles are ever evolving, that we will update and
+refine as we learn and grow.
+
+## Values
+
+Specific beliefs
+
+-   *How* we work is as important as *what* we make or do
+-   Work that is visible and open is easier to participate in and
+    contribute to
+-   Code itself is as valuable as working software
+-   Working software without documentation is not as useful without
+    documentation
+-   Inclusive and accessible design make software and documentation more
+    usable and impactful
+-   Openness and transparency lead to better collaboration and products
+
+## Principles
+
+We follow these specific principles that are influenced by our values.
+We will:
+
+-   document as many processes and activities as we are able to by
+    following a [documentation-first
+    approach](https://community.seedcase-project.org/guides/documentation-first)
+-   work and communicate openly (though privately where necessary)
+-   use open licenses for all our outputs
+-   design documentation, training material, and software for people who
+    are not computationally technical
```

Comment:
design documentation, training material, and software with non-technical users in mind

Reply:
I originally had that exact sentence, but I wanted to avoid the use of "user" in this particular instance (because I want to focus on people) and because "technical" can mean many things.

Reply:
Computer literate then, and people ? computationally technical is a bit of a mouthful, isn't it?